crates/polars-io/src/csv/write/write_impl/serializer.rs

```diff
@@ -264,7 +264,7 @@ fn decimal_serializer(array: &PrimitiveArray<i128>, scale: usize) -> impl Serial
 fn callback_serializer<'a, T: NativeType, const QUOTE_NON_NULL: bool>(
     array: &'a PrimitiveArray<T>,
     mut callback: impl FnMut(T, &mut Vec<u8>) + 'a,
-) -> impl Serializer + 'a {
+) -> impl Serializer<'a> {
```

Comment:
Not 100% sure this is equivalent? 🤔

Reply:
Not exactly. Might not be an issue though. Why was it needed now?

Reply:
There's a new lint that doesn't allow eliding lifetimes if the lifetime is named elsewhere in the function definition, so `Serializer` has to explicitly become `Serializer<'a>`. Apparently the `+ 'a` is no longer needed.